doc/usermanual/preferences/gui_options.xml

```diff
@@ -16,280 +16,284 @@
     These options control the look and feel of darktable.
   </para>
 
-  <bridgehead renderas="sect4">width of the side panels in pixels</bridgehead>
+  <sect2 id="gui_options_import">
 
-  <para>
-    This controls the size of side panels in pixels. Side panels are found left and right to the
-    center view (default 350).
```

Comment:
This controls the size of the side panels (in pixels). Side panels are found tot he left and right of the
    center view (default 350).